iso_to_ipxe

```diff
@@ -45,37 +45,43 @@ fi
 
 mkdir -p $IPXE_DIR
 
-echo Configuring IPXE 
+echo Configuring IPXE
 
 cat > $IPXE_DIR/ipxe << EOF
-#!ipxe                                                                                                                                                                                    
+#!ipxe
 set live_url ${BASE_URL}
 initrd --name main ${live_url}/initrd.img
 kernel \${live_url}/vmlinuz initrd=main ignition.config.url=\${live_url}/config.ign coreos.live.rootfs_url=\${live_url}/rootfs.img ${KERNEL_OPTS}
-boot       
+boot
 EOF
 
 export PXE_IMAGES=`isoinfo -i $IMAGE -f |grep -i images/pxeboot`
 for img in $PXE_IMAGES; do
-  export name=`basename ${img,,} | sed 's/\;1//' | sed 's/\.$//'`
+  export name=`basename ${img} | tr '[:upper:]' '[:lower:]' | sed 's/\;1//' | sed 's/\.$//'`
   echo extracting $name
   isoinfo -i $IMAGE -x $img > $IPXE_DIR/$name
 done
- 
+
 echo writing custom user ignition
 
 # Check if ISO is gzipped or xzed
 IGNITION_IMG_TYPE=$(isoinfo -i $IMAGE -x '/IMAGES/IGNITION.IMG;1' | file - | awk '{print $2}')
 
+tmp_ignition=$(mktemp)
+__cleanup(){
+  rm -f $tmp_ignition
+}
+trap __cleanup EXIT
+
 if [ $IGNITION_IMG_TYPE == "gzip" ]
 then
-  isoinfo -i $IMAGE -x '/IMAGES/IGNITION.IMG;1' | gunzip | cpio -i --to-stdout > $IPXE_DIR/config.ign
```

Comment:
Sure, the temp file is needed because some cpio versions dont have the options `--to-stdout` or `-D` (output directory)

So my solution was to change to the $IPXE_DIR and run cpio there

Since we cannot be sure if $IMAGE is a relative path, we cannot change to $IPXE_DIR and run the full chain of commands there

Another solution I just thought now is to extract config.ign to the current path (probably where iso_to_ipxe resides) and then move the file to $IPXE_DIR